src/_index.js

```diff
@@ -69,8 +51,8 @@ const defaults = {
  */
 class EasySelect{
     constructor(el, options){
-        this.classes = {...classes};
-        this.atts = {...atts};
+        this.classes = {...CLASSES};
+        this.atts = {...ATTRS};
```

Comment:
What's the point of creating `configs.js` while you still keep these variables: `this.classes`, `this.atts`?